docs/config.md

```diff
@@ -1,211 +1,466 @@
+   not present in the config, the app will disable all rageshake functionality. Set to `"https://element.io/bugreports/submit"` to submit
+   rageshakes to us, or use your own rageshake server.
+2. `uisi_autorageshake_app`: If a user has enabled the "automatically send debug logs on decryption errors" flag, this option will be sent
+   alongside the rageshake so the rageshake server can filter them by app name. By default, this will be `"element-web"`, as with any other
+   rageshake submitted by the app.
+
+   If you are using the element.io rageshake server, please set this to `"element-auto-uisi"` so we can better filter them.
+
+If you would like to use [Sentry](https://sentry.io/) for rageshake data, add a `sentry` object to your config with the following values:
+
+1. `dsn`: The Sentry [DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/).
+2. `environment`: Optional [environment](https://docs.sentry.io/product/sentry-basics/environments/) to pass to Sentry.
+
+For example:
+
+```json
+{
+   "sentry": {
+      "dsn": "dsn-goes-here",
+      "environment": "production"
+   }
+}
+```
+
+## Integration managers
+
+Integration managers are embedded applications within Element to help the user configure bots, bridges, and widgets. An integration manager
+is a separate piece of software not typically available with your homeserver. To disable integrations, leave the options defined here out of
+your config.
+
+1. `integrations_ui_url`: The UI URL for the integration manager.
+2. `integrations_rest_url`: The REST interface URL for the integration manager.
+3. `integrations_widgets_urls`: A list of URLs the integration manager uses to host widgets.
+
+If you would like to use Scalar, the integration manager maintained by Element, the following options would apply:
+
+```json
+{
+   "integrations_ui_url": "https://scalar.vector.im/",
+   "integrations_rest_url": "https://scalar.vector.im/api",
+   "integrations_widgets_urls": [
+      "https://scalar.vector.im/_matrix/integrations/v1",
+      "https://scalar.vector.im/api",
+      "https://scalar-staging.vector.im/_matrix/integrations/v1",
+      "https://scalar-staging.vector.im/api",
+      "https://scalar-staging.riot.im/scalar/api"
+   ]
+}
+```
+
+## Administrative options
+
+If you would like to include a custom message when someone is reporting an event, set the following Markdown-capable field:
+
+```json
+{
+   "report_event": {
+      "admin_message_md": "Please be sure to review our [terms of service](https://example.org/terms) before reporting a message."
+   }
+}
+```
+
+To add additional "terms and conditions" links throughout the app, use the following template:
+
+```json
+{
+   "terms_and_conditions_links": [
+      { "text": "Code of conduct", "url": "https://example.org/code-of-conduct" }
+   ]
+}
+```
+
+## Analytics
+
+Analytics are currently possible with two systems: `piwik` (matomo; deprecated) and `posthog` (preferred). When these configuration
```

Comment:
let's make posthog first and add a strike-through for piwik to discourage its usage?